assets/css/podcasting-onboarding.scss

```diff
@@ -2,10 +2,6 @@
 	box-sizing: border-box;
 }
 
-#wpcontent {
-	padding-left: 0;
-}
```

Comment:
Minor: Need to fix this space issue in the onboarding header.

![Screenshot 2023-08-18 at 12 44 47 PM](https://github.com/10up/simple-podcasting/assets/10613171/6987607d-f04b-451a-a34a-259d8950e20f)